spec/models/manageiq/providers/hawkular/middleware_manager_spec.rb

```diff
@@ -1,4 +1,27 @@
 describe ManageIQ::Providers::Hawkular::MiddlewareManager do
+  subject { described_class.new }
+
+  it "includes server operations" do
```

Comment:
Addittionally to the interface inclusion tests, I would test each one to ensure that the actual `run` is done. But I don't know if that is over-testing.